test-data/unit/lib-stub/typing.py

```diff
@@ -61,6 +62,9 @@ class Awaitable(Generic[T]):
     @abstractmethod
     def __await__(self) -> Generator[Any, Any, T]: pass
 
+class AwaitableGenerator(Generator[T, U, V], Awaitable[V], Generic[T, U, V, S]):
```

Comment:
Why is there an `AwaitableGenerator` in the test data `typing` stub when there isn't one in the real `typing` module?